components/pokemonapp.brs

```diff
@@ -30,4 +28,21 @@ function createPokeList(listRow as Object)
     newPokemonContent.appendChild(listRow)
     m.pokeList.content = newPokemonContent
     m.pokeList.setFocus(true)
-end function
\ No newline at end of file
+    observePokeList()
+end function
+
+function observePokeList()
+    m.pokeList.observeField("rowItemFocused","parsePokeData")
```

Comment:
You can rename `parsePokeData` in something more suggestive. Maybe `onRowItemFocused` or `onPokemonFocused`